src/libraries/Microsoft.PowerFx.Core/Binding/BindingConfig.cs

```diff
@@ -19,10 +20,13 @@ internal class BindingConfig
 
         public bool UseThisRecordForRuleScope { get; }
 
-        public BindingConfig(bool allowsSideEffects = false, bool useThisRecordForRuleScope = false)
+        public CultureInfo CultureInfo { get; }
+
+        public BindingConfig(bool allowsSideEffects = false, bool useThisRecordForRuleScope = false, CultureInfo locale = null)
```

Comment:
this is potentially breaking. 
May need another overload.

Reply:
Removed.